lighthouse-core/report/v2/renderer/report-ui-features.js

```diff
@@ -64,7 +64,7 @@ class ReportUIFeatures {
    * Fires a custom DOM event on target.
    * @param {string} name Name of the event.
    * @param {!Node=} target DOM node to fire the event on.
-   * @param {Object<{detail: Object<string, *>}>=} detail Custom data to include.
+   * @param {*=} detail Custom data to include.
```

Comment:
hehe, that's what it really is! https://dom.spec.whatwg.org/#dictdef-customeventinit
```idl
dictionary CustomEventInit : EventInit {
  any detail = null;
};
```